### PR TITLE
Key pair generation

### PR DESCRIPTION
**Pull request by bakurits**
Added project structure
Implemented key pair generation/modification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Plugin Starter Template [![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
+# Anonymous Plugin [![CircleCI branch](https://img.shields.io/circleci/project/github/bakurits/mattermost-plugin-anonymous/key-pair-generation.svg)](https://circleci.com/gh/bakurits/mattermost-plugin-anonymous) [![codecov](https://codecov.io/gh/bakurits/mattermost-plugin-anonymous/branch/key-pair-generation/graph/badge.svg)](https://codecov.io/gh/bakurits/mattermost-plugin-anonymous)
```
Comment (Gvantsats) [inline README.md:1]: links are incorrect

File: server/anonymous/anonymous.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		MattermostUserID: a.actingMattermostUserID,
+		PublicKey:        publicKey,
+	})
+}
+
+//GetPublicKey get public key from plugin's KeyValue Store
+func (a *anonymous) GetPublicKey() (crypto.PublicKey, error) {
+	user, err := a.LoadUser(a.actingMattermostUserID)
+	if err != nil {
+		return nil, err
```
Comment (iomodo) [inline server/anonymous/anonymous.go:64]: wrap the error with `errors.Wrapf(err, "Error while loading a user %s", a.actingMattermostUserID)`

File: server/command/command.go [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+// NewHandler returns new Handler with given dependencies
+func NewHandler(args *model.CommandArgs, a anonymous.Anonymous) Handler {
+	return newCommand(args, a)
+}
+
+func (c *command) Handle(args ...string) (*model.CommandResponse, error) {
+	ch := c.handler
+	if len(args) == 0 || args[0] != "/anonymous" {
```
Comment (iomodo) [inline server/command/command.go:69]: use const

File: server/command/command.go [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+func (c *command) responseRedirect(redirectURL string) *model.CommandResponse {
+	return &model.CommandResponse{
+		GotoLocation: redirectURL,
+	}
+}
+
+// GetSlashCommand returns command to register
+func GetSlashCommand() *model.Command {
+	return &model.Command{
+		Trigger:          "anonymous",
```
Comment (iomodo) [inline server/command/command.go:140]: const here

**Pull request merged**